pages/common/mpg321.md

```diff
@@ -0,0 +1,32 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
+> mpg321 was written to be a drop-in replacement for the (by-then) non-free mpg123 player.
+
+- Play an audio source (on repeat) exactly N=3 times:
+
+`mpg321 -l {{3}} {{ file(s) | URL }}`
```

Comment:
`{{ file(s) | URL }}` should be something like `{{path/to/file|url}}` and folder should be replaced with directory in other examples. Perhaps other maintainers have a better idea how to handle this.